doc/conf.py

```diff
@@ -438,20 +438,21 @@
 
 # Configuration for intersphinx
 intersphinx_mapping = {
-    'cf_units': ('https://cf-units.readthedocs.io/en/latest/', None),
+    'cf_units': ('https://cf-units.readthedocs.io/en/stable/', None),
     'cftime': ('https://unidata.github.io/cftime/', None),
     'esmvalcore':
     (f'https://docs.esmvaltool.org/projects/ESMValCore/en/{rtd_version}/',
      None),
     'esmvaltool': (f'https://docs.esmvaltool.org/en/{rtd_version}/', None),
+    "esmpy": ("https://earthsystemmodeling.org/esmpy_doc/release/latest/html/",
```

Comment:
```suggestion
    'esmpy': ('https://earthsystemmodeling.org/esmpy_doc/release/latest/html/',
```

Just to be consistent with the other entries

Reply:
Done